spec/models/book_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
i opened https://github.com/ajkamel/cover/pull/6 in case we want to change any of the settings, i prefer single.